IDE/ECLIPSE/DEOS/README.md

```diff
@@ -0,0 +1,210 @@
+
```

Comment:
I don't see where the IDE/ECLIPSE folder is being included in an .am file at the IDE/ level.